### PR TITLE
update symlink to mongo_3_2

### PR DESCRIPTION
**Pull request by icarrr**
MongoDB 2.6 reached End of Life (EOL)

Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @icarrr! I've created [OSPR-3855](https://openedx.atlassian.net/browse/OSPR-3855) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://open.edx.org/wp-content/uploads/2019/01/individual-contributor-agreement.pdf)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

**Comment by natabene** [comment]
@icarrr Thank you for the contribution! When you have a chance, could you please send in a signed Contributor Agreement mentioned in the earlier comment? You only need to do this once and we can then review all your future code contributions. Thanks

**Comment by icarrr** [comment]
@natabene Okay, the document has been sent to email legal@edx.org. Thanks

**Comment by natabene** [comment]
@icarrr I checked with the legal team, and look like they are waiting on you to provide signature.

**Comment by icarrr** [comment]
@natabene Okay, I've sent the document to legal@edx.org

**Comment by nedbat** [comment]
@icarrr for our review, is this change based on a similar change on master?

**Comment by icarrr** [comment]
@nedbat yes, for mongoDB changes, this change based on a similar change on master

**Comment by nedbat** [comment]
Can you provide a link to that change for our comparison?

**Comment by icarrr** [comment]
@nedbat Which link do you mean?

**Comment by nedbat** [comment]
Is there a pull request for the change on master? Or even a commit you could link to on GitHub?  Ficus isn't supported any longer, so we try to limit the effort we put into it. If you can direct me to the corresponding change on master, it will be easier to review this change.

**Comment by icarrr** [comment]
@nedbat I got it from here [mongo_3_2](https://github.com/edx/configuration/tree/master/playbooks/roles/mongo_3_2) & commit https://github.com/edx/configuration/commit/a2122c1adeed3d51c5ef4e1fc7f66440a09160d6

File: util/install/native.sh [hunk 1]
```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```
Comment (nedbat) [inline util/install/native.sh:1]: I don't want to add this file to Ficus. It changes the installation flow. Did you mean to include it in this pull request?

**Review by nedbat — COMMENTED** [review]
But there are more changes here than just that directory and that version switch.  Ficus is unsupported.  I don't know how to verify this change without a lot of effort.  Is there another way we can support Ficus installers?

File: util/install/native.sh (hunk 1 above)
Comment (icarrr) [reply util/install/native.sh:1]: I don't want that to be included in a pull request, just a change to the mongodb role.

Comment (nedbat) [reply util/install/native.sh:1]: OK, you need to get this pull request into the state you want.

Comment (icarrr) [reply util/install/native.sh:1]: How? Do I need to delete the file or others?

**Comment by natabene** [comment]
@icarrr Thank you for joining the Open edX community! We have added your user to our records and can now review your code going forward.

**Comment by nedbat** [comment]
@icarrr Perhaps you can find other users of Ficus who can help get this pull request into shape.  It would help me if you could have a few people vouch for the correctness of the change.
https://openedx.org/getting-help has links to places where support discussions happen.

I'll close this for now.  Feel free to re-open it when you are ready.

**Comment by openedx-webhooks** [comment]
@icarrr Even though your pull request wasn’t merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/configuration&entry.867055334=https://github.com/edx/configuration/pull/5421&entry.1484655318=https://github.com/icarrr&entry.752974735=2019-09-25+13:44&entry.1917517419=2019-10-16+17:58&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**